rpa-builder/6.4/modules/ROOT/pages/toolbox-network-rest-call.adoc

```diff
@@ -92,6 +92,18 @@ A path parameter that uniquely identifies the resource.
 +
 The Xpath expression to the property of the response object. For example, `/root/access_token/text()`.
 
+* *Timing* 
+
+** *Timeout*
++
+The time in seconds the activity waits. Settings this value to `0` forces the activity to wait indeterminately. 

```

Comment:
**issue:** 0 will always lead to an execution error. There is no "indeterminate" setting.